src/__tests__/roadwork.ts

```diff
@@ -0,0 +1,46 @@
+import { MongoMemoryServer } from 'mongodb-memory-server';
+import mongoose from 'mongoose';
+
+import { createRoadwork, findRoadwork } from '../models/Roadwork';
+
+const mongod = new MongoMemoryServer();
+
+const databaseName = 'test_roadwork';
+
+const name = 'name1';
+const description = 'description1';
+
+describe('testing roadwork', () => {
+  beforeAll(async () => {
+    const uri = await mongod.getUri(databaseName);
+
+    await mongoose.connect(uri, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+      keepAlive: true,
+    });
+  });
+
+  it('creates a roadwork database object', async () => {
+    const result = await createRoadwork(null, {
+      name,
+      description,
+    });
+
+    expect(result.name).toEqual(name);
+    expect(result.description).toEqual(description);
+  });
+
+  it('finds the previously created roadwork', async () => {
+    const result = await findRoadwork();
+    expect(result).toHaveLength(1);
+    expect(result[0]?.name).toEqual(name);
+    expect(result[0]?.description).toEqual(description);
+  });
+
+  afterAll(async () => {
+    await mongoose.connection.dropDatabase();
+    await mongoose.connection.close();
+    mongod.stop();
+  });
+});
```

Comment:
have you found a jest file naming convention? are there best practices you followed? I know naming files with suffix `-test` or `.test` which can have advantages in using the development environment. for example in VSCode when jumping to a file `App` the `App.js` wanted appears in first place, while the `App.test.js` comes later.

<img width="366" alt="Bildschirmfoto 2021-03-17 um 12 44 01" src="https://user-images.githubusercontent.com/1942953/111462497-cc392d80-871e-11eb-9338-1d61e2cfdfa2.png">

With same file name it can be a bit more difficult to find the right one. A file in the root directory can be more easy as the file path is short, but with longer file path it is not directly clear, which file is the src one and which the __test__ one.

<img width="285" alt="Bildschirmfoto 2021-03-17 um 12 46 54" src="https://user-images.githubusercontent.com/1942953/111462671-fd196280-871e-11eb-9c9f-9dce4a3bcd91.png">
<img width="213" alt="Bildschirmfoto 2021-03-17 um 12 48 24" src="https://user-images.githubusercontent.com/1942953/111462743-14585000-871f-11eb-85da-3f707d8b3988.png">

Reply:
see above